EDDI/MainWindow.xaml.cs

```diff
@@ -565,43 +576,85 @@ private void ChangeLog_Click(object sender, RoutedEventArgs e)
+            try
             {
-                try
+                string IssueLogDir = Constants.DATA_DIR + @"\logexport\";
+                string IssueLogFile = IssueLogDir + @"eddi_issue.log";
+                string DesktopPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + @"\eddi_issue.zip";
+
+                lock (logLock)
                 {
                     progress.Report("");
-                    client.UploadFile("http://api.eddp.co/log", Constants.DATA_DIR + @"\\eddi.log");
-                    progress.Report("done");
-                    if (false) // temporarily disabled while we fix #91
+
+                    // Create a temporary issue log file, delete any remnants from prior issue reporting
+                    Directory.CreateDirectory(IssueLogDir);
+                    File.Create(IssueLogFile);
+                    File.Delete(DesktopPath);
+
+                    // Use regex to isolate DateTimes from the string
+                    Regex recentLogsRegex = new Regex(@"^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})");
+
+                    var log = File.ReadAllLines(Constants.DATA_DIR + @"\\eddi.log");
+                    double elapsed_time = 0;
+
+                    foreach (string line in log)
                     {
                         try
                         {
-                            File.Delete(Constants.DATA_DIR + @"\\eddi.log");
+                            // Parse log file lines so that we can examine DateTimes
+                            string linedatestring = recentLogsRegex.Match(line).Value;
+                            DateTime linedate = DateTime.Parse(linedatestring);
+                            elapsed_time = (DateTime.UtcNow - linedate).TotalHours;
+
+                            // Fill the issue log with log lines from the most recent hour only
+                            if (elapsed_time < 1)
+                            {
+                                using (StreamWriter file = new StreamWriter(IssueLogFile, true))
+                                {
+                                    file.WriteLine(line);
+                                }
+                            }
                         }
-                        catch (Exception ex)
+                        catch (Exception)
                         {
-                            Logging.Error("Failed to delete file after upload", ex);
+                            // Do nothing, adding to the debug log creates a feedback loop
                         }
                     }
                 }
-                catch (Exception ex)
-                {
-                    progress.Report("failed");
-                    Logging.Error("Failed to upload log", ex);
-                }
+                // Copy the issue log & zip it to the desktop so that it can be added to the Github issue
+                ZipFile.CreateFromDirectory(IssueLogDir, DesktopPath);
```

Comment:
This is failing for me with exception "System.IO.IOException: The process cannot access the file 'C:\Users\Richard Buckle\AppData\Roaming\EDDI\logexport\eddi_issue.log' because it is being used by another process."

Reply:
I am not getting this error. What do you believe is the source?